### ISSUE TITLE
IE10 - fuzzy/blurry Font and URL is displaying in footer

### ISSUE THREAD

**Issue by CRajiv** [mozilla/pdf.js#6247]
First page alone looks good and other page fonts are fuzzy/blurry in IE 10. PDF pages also printing with Browser URL and Page numbers.

```
    PDFJS.getDocument(pdfUrl).then(function (pdf) {
        var currentPage = 1;
        function getPages() {
            pdf.getPage(currentPage).then(function (page) {
                var PRINT_OUTPUT_SCALE = 2,
                    viewport = page.getViewport(PRINT_OUTPUT_SCALE),
                    canvas = document.createElement('canvas'),
                    canvasWrapper = document.createElement('div');
                canvasWrapper.classList.add('canvasWrapper');

                canvas.setAttribute('class', 'canvaspdf');
                canvas.id = 'canvas' + currentPage + '-' + pdfindex;

                canvas.width = Math.floor(viewport.width);
                canvas.height = Math.floor(viewport.height);

                canvasWrapper.appendChild(canvas);
                document.getElementsByTagName('html')[0].setAttribute('mozNoMarginBoxes', '');

                var context = canvas.getContext('2d'),
                    renderContext = {
                        canvasContext: context,
                        viewport: viewport,
                        intent: 'print'
                    };


                page.render(renderContext).then(function (page) {
                    var printContainer = document.getElementById('printContainer');
                    printContainer.appendChild(canvasWrapper);
                    if (currentPage < pdf.numPages) {
                        currentPage++;
                        getPages();
                    } else {
                        next();
                    }
                });
            });
        }

        if (currentPage <= pdf.numPages) {
            getPages();
        }
    }, function () {
        alert("failed");
    });
```

CSS

```
.canvaspdf 
{
display: block;
height: 1186px;
width: 917px;
margin: 0;
padding: 0;
position: static;
visibility: visible;
}

@page{
    size: letter portrait;
    margin: 0mm;
 }

  .canvasWrapper{
      page-break-inside:avoid;
 }
 canvas{
     page-break-inside:avoid;
  }
```

**Comment by timvandermeij** [comment]
We need an example PDF to verify the issue. Without that it is really unclear what the problem is.

**Comment by CRajiv** [comment]
OS: Windows 7
Browser : IE10
![ie_print_issues](https://cloud.githubusercontent.com/assets/13448382/8869590/26467462-3203-11e5-9115-56f967905b06.png)

**Comment by CRajiv** [comment]
@timvandermeij still we are facing this issue.

**Comment by timvandermeij** [comment]
Could be partially fixed by #6160. We cannot do anything about the page numbers and URL being printed. AFAIK JavaScript does not have access to those browser-specific print settings.

**Comment by timvandermeij** [comment]
Closing since IE10 is no longer supported since PDF.js 2.0.
